.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+  "arrowParens": "avoid",
```

Comment:
Personally, I would also go with the default here as I tend to agree with their rationale:
> At first glance, avoiding parentheses may look like a better choice because of less visual noise. However, when Prettier removes parentheses, it becomes harder to add type annotations, extra arguments or default values as well as making other changes. Consistent use of parentheses provides a better developer experience when editing real codebases, which justifies the default value for the option.
https://prettier.io/docs/en/options.html#arrow-function-parentheses

Reply:
I think the main argument here is type annotations, which we don't have. I personally prefer less visual noise, but of course whatever is preferred in the project goes.